css/gallerybutton.css

```diff
@@ -143,3 +143,10 @@ http://davidwalsh.name/css-flip*/
 	-moz-transform: rotateX(180deg);
 	transform: rotateX(180deg);
 }
+
+#controls .right .selectedActions a {
+	opacity: 1;
+}
+#controls .right .selectedActions .button .hidden {
+	display: none;
```

Comment:
`.hidden` class is already defined in `global.scss`, so it shouldn't be needed here.

Reply:
I would love not to add complexity in stylesheets, but we need this since server.css is adding a rule that takes precence over .hidden and breaks UI when clicking on download button : 
`
button > span[class^="icon-"], .button > span[class^="icon-"], button > span[class*=" icon-"], .button > span[class*=" icon-"] {
    display: inline-block;
}
`
The solution would be to change server or add an important mark on hidden within Gallery, but it may have side effects while my solution is kind of a workaround but does not have any side effects. What do you think?

Reply:
what if you give `.hidden` to the `button` element instead of the icon itself?

Reply:
An image is worth a thousand words : this is what I have without my CSS rule when clicking on download button : ![loading-download](https://user-images.githubusercontent.com/38929/33131896-46bdfbd6-cf98-11e7-8359-3a26a7b8cdeb.png)

I am replacing the "download" icon inside of the button by a "loading" icon. Thus hiding the button does not work, unless I completely change the logic (which I would be happy to do if needed).